dashboard/app/models/peer_review.rb

```diff
@@ -175,7 +179,7 @@ def self.get_peer_review_summaries(user, script)
             reviews.size < script.peer_reviews_to_complete &&
             PeerReview.get_potential_reviews(script, user).any?
           reviews << {
-            status: 'not_started',
+            status: LEVEL_STATUS.not_tried,
```

Comment:
This change from 'not_started' to 'not_tried' is intentional. (It should never have been not_started here)